lib/scenes/help_scene.cpp

```diff
@@ -0,0 +1,76 @@
+#include "scenes/help_scene.hpp"
+
+#include "entities/entity_factory.hpp"
+#include "scenes/data/menu/image.hpp"
+
+HelpScene::HelpScene(EntityFactory& factory, BrickEngine& engine, GameController& game_controller) : Menu(factory, engine, WIDTH, HEIGHT), game_controller(game_controller) { }
+
+void HelpScene::performPrepare() {
+    entity_components = std::make_unique<std::vector<EntityComponents>>();
+
+    // Images
+    Image keyboard = Image();
+    keyboard.texture_path = "menu/keyboard.png";
+    keyboard.alpha = 255;
+    keyboard.x = 1320;
+    keyboard.y = 180;
+    keyboard.x_scale = 1160;
+    keyboard.y_scale = 338;
+    this->images.push_back(keyboard);
+
+    Image controller = Image();
+    controller.texture_path = "menu/controller.png";
+    controller.alpha = 255;
+    controller.x = 1560;
+    controller.y = 600;
+    controller.x_scale = 552;
+    controller.y_scale = 386;
+    this->images.push_back(controller);
```

Comment:
The help function works great, therefore  this is more of a 'UI' suggestion rather than a functional comment. I would place the text for the controller in the top right corner, and put the controller left of it. The keyboard placed below and aligned with the controller and the explanation text. This is a suggestion to not have all the text so close to each other and make it (or at least I think) more readable when you just want to go through them quickly.